_posts/2022-01-10-reassign-partitions.md

```diff
@@ -319,11 +322,322 @@ Now we can see if the preferred leader is now changed or not
         Topic: my-topic-two     Partition: 0    Leader: 0       Replicas: 1,0,2 Isr: 0,1,2
         Topic: my-topic-two     Partition: 1    Leader: 2       Replicas: 1,2,3 Isr: 2,3,1
         Topic: my-topic-two     Partition: 2    Leader: 3       Replicas: 2,3,4 Isr: 2,3,4
+```
+
+## Example 3: Changing the log dirs 
+
+Lets see how we can move a partition to use a certain JBOD volume.
+
+Let say we have deployed a Kafka Resource with JBOD storage and deployed a topic with `spec.replicas` as 4.
 
+Now our second task is to check which `logdir` are currently being used by Strimzi. You can use the `kafka-log-dir.sh` tool to check this
+
+```shell
+kubectl exec -n myproject -ti my-pod /bin/bash -- bin/kafka-log-dirs.sh --describe --bootstrap-server my-cluster-kafka-bootstrap:9092  --broker-list 0,1,2,3 --topic-list my-topic-two
+```
+
+You will get a `json` which represents the log directories being used by the brokers
+
+```json
+[
+  {
+    "broker": 0,
+    "logDirs": [
+      {
+        "logDir": "/var/lib/kafka/data-0/kafka-log0",
+        "error": null,
+        "partitions": [
+          {
+            "partition": "my-topic-two-1",
+            "size": 0,
+            "offsetLag": 0,
+            "isFuture": false
+          },
+          {
+            "partition": "my-topic-two-2",
+            "size": 0,
+            "offsetLag": 0,
+            "isFuture": false
+          }
+        ]
+      },
+      {
```

Comment:
It might be clearer if you elide the rest of this JSON as it's pretty big, and it's only a small part which we will end up changing. It's not legal JSON syntax, but I think using `// ... and similar for the other logdirs and brokers` would be much clearer for the reader.